pkg/applicationserver/io/io.go

```diff
@@ -174,3 +176,27 @@ func CleanDownlinks(items []*ttnpb.ApplicationDownlink) []*ttnpb.ApplicationDown
 	}
 	return res
 }
+
+// PipeSubscription pipes the output of a Subscription to the provided handler.
+func PipeSubscription(ctx context.Context, ts component.TaskStarter, name string, sub *Subscription, submit func(context.Context, interface{}) error) {
```

Comment:
Nit: I think the more idiomatic way would be to make this a method of Subscription right? 
```
Subscription.Pipe(xxxx)
```